hail/src/main/scala/is/hail/expr/ir/lowering/LowerToCDA.scala

```diff
@@ -10,15 +15,56 @@ object LowerToCDA {
 
     // Slightly simpler to replace RelationalRefs in a second linear pass,
     // since we'd have to embed the rewrite in many places otherwise
-    RewriteBottomUp(lower(ir, typesToLower, ctx, r, Seq()), {
+    RewriteBottomUp(lower(ir, typesToLower, ctx, r, Map()), {
```

Comment:
Do you need the second pass any more? Doesn't `lower` now leave no `RelationalRef`s`

Reply:
oh, yeah, forgot to remove this.